tracer/src/Datadog.Trace/Tracer.cs

```diff
@@ -10,6 +10,7 @@
 using Datadog.Trace.Agent;
 using Datadog.Trace.ClrProfiler;
 using Datadog.Trace.Configuration;
+using Datadog.Trace.Logging;
```

Comment:
Unused, if the log below is unused
```suggestion
using Datadog.Trace.Logging;
```